qiita_ware/ebi.py

```diff
@@ -105,21 +86,44 @@ class EBISubmission(object):
     Parameters
     ----------
     preprocessed_data_id : str
+
+    Raises
+    ------
+    EBISumbissionError
+        - If the action is not in EBISubmission.valid_ebi_actions
```

Comment:
It is not required by numpy doc, but is not explicitly prohibited. Happy to leave as it is unless somebody else feels strong about it.